lib/visualRegressionTracker.ts

```diff
@@ -89,17 +96,17 @@ export class VisualRegressionTracker {
     }
   }
 
-  async track(test: TestRun) {
-    const result = await this.submitTestResult(test);
+  async track(test: TestRun): Promise<TestRunResult> {
```

Comment:
Codacy found an issue: [Async method 'track' has a complexity of 5. Maximum allowed is 4.](https://app.codacy.com/gh/Visual-Regression-Tracker/sdk-js/pullRequest?prid=6318104)